src/main/scala/com/gu/sfl/model/model.scala

```diff
@@ -49,8 +49,10 @@ case class SavedArticles(version: String, articles: List[SavedArticle]) extends
   override def advanceVersion: SavedArticles = copy(version = nextVersion)
   @JsonIgnore
   lazy val numberOfArticles = articles.length
+  //NewestFirst
   def ordered: SavedArticles = copy(articles = articles.sorted.reverse)
-  def deduped = copy( articles = articles.groupBy(_.id).map(_._2.sorted.head).toList.sorted.reverse)
+  def deduped: SavedArticles = copy( articles = articles.groupBy(_.id).map(_._2.sorted.head).toList.sorted.reverse )
+  def persitable(limit: Int) = copy( articles = articles.sorted.reverse.takeRight(limit)  )
```

Comment:
It's already reversed by the Ordering?

Reply:
Yeah its a different use case: The apps expect the array to be oldest first - whereas here we want to take the most recent n, but still render the list oldest first

Reply:
Sorry to pester on this point, but should this be
```articles.sorted.takeRight(limit).reverse```